test/integration/variable_test.rb

```diff
@@ -79,4 +79,10 @@ def test_hash_with_default_proc
   def test_multiline_variable
     assert_equal 'worked', Template.parse("{{\ntest\n}}").render!('test' => 'worked')
   end
+
+  def test_global_options_on_variable
+    template = Template.parse(%({{test}}), global_filters: ['truncate: 5'])
```

Comment:
@dylanahsmith Thanks for the feedback. Can you elaborate a little bit more on what you mean by using a Proc instead of using liquid syntax?

I'm guessing you would want these filters applied at some other point in the process than I currently have implemented? aka during parsing.